exporter/metric/internal/statsd/conn.go

```diff
@@ -34,8 +34,8 @@ import (
 )
 
 type (
-	// Config supports common options that apply to statsd exporters.
-	Config struct {
+	// Options supports common options that apply to statsd exporters.
+	Options struct {
```

Comment:
I think this is an unrelated remark.

I thought we had moved toward the practice of using "Config" as the structure name, and Option as a functional argument (`func(*Config)`), and Options as a `[]Option`.  See api/trace `StartConfig` and `StartOption`, for example.

That's why I prefer this struct be called Config.

Reply:
See https://github.com/open-telemetry/opentelemetry-go/pull/369

Reply:
The goal of this was to standardize the patterns for all Exporter options/config. Prometheus and stdout were using options, statsd was using config...